cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBrokerConfigurationBuilder.java

```diff
@@ -128,7 +130,10 @@ public KafkaBrokerConfigurationBuilder withCruiseControl(String clusterName, Cru
                                                              String replicationFactor, String minInSyncReplicas)   {
         if (cruiseControl != null) {
             printSectionHeader("Cruise Control configuration");
-            writer.println(CruiseControlConfigurationParameters.METRICS_TOPIC_NAME + "=strimzi.cruisecontrol.metrics");
+            Map<String, Object> config = cruiseControl.getConfig();
+            String metricsTopicName = Optional.ofNullable(config.get(CruiseControlConfigurationParameters.CRUISE_CONTROL_METRIC_REPORTER_TOPIC_NAME.getValue()))
+                            .orElse(CruiseControlConfigurationParameters.DEFAULT_METRIC_REPORTER_TOPIC_NAME).toString();
+            writer.println(CruiseControlConfigurationParameters.METRICS_TOPIC_NAME + "=" + metricsTopicName);
```

Comment:
Can't you use `getOrDefault` instead of `Optional.ofNullable`? I think it would be more readable. Also, IIRC, the `ofNullable` does not catch NPEs => so you need to check if `cruiseConfig.getConfig()` is not null.

Reply:
Good point about the null config, I've changed it and made use of getOrDefault, see what you think